test/query.js

```diff
@@ -58,5 +59,16 @@ describe('Browser client', () => {
       done: () => done(),
     });
   })
+
+  it('table schema (table exists)', () => {
+    return client.table(TABLE)
+      .then(result => result.object.should.be.an.object)
```

Comment:
I would appreciate a more in-depth test that checks the structure of the object returned. It makes it easier to know how to use the function.

Reply:
The object is an array, I could change it to check array but the content would change therefore I decided no check more in-depth.

`result.object` returns something like this:
```javascript
[
  {
    "fieldName": "eventdate",
    "type": "timestamp"
  },
  {
    "fieldName": "clientIpAddress",
    "type": "ip4"
  },
  {
    "fieldName": "timestamp",
    "type": "str"
  },
  {
    "fieldName": "method",
    "type": "str"
  },
  {
    "fieldName": "uri",
    "type": "str"
  },
  {
    "fieldName": "protocol",
    "type": "str"
  },
  {
    "fieldName": "statusCode",
    "type": "int8"
  },
  {
    "fieldName": "bytesTransferred",
    "type": "int8"
  },
  {
    "fieldName": "referralUri",
    "type": "str"
  },
  {
    "fieldName": "userAgent",
    "type": "str"
  },
  {
    "fieldName": "cookie",
    "type": "str"
  },
  {
    "fieldName": "timeTaken",
    "type": "int8"
  }
]
```

Reply:
Hmmm, then I think it would be enough to check `result.object.length` or something like that?

Reply:
With the new tests way I checked the whole content using a results mock.